packages/libphonenumber/libphonenumber/utils.js

```diff
@@ -2,44 +2,61 @@
 * Our wrapper around google's libphonenumber.
 */
 var phonenumber = require('libphonenumber/libphonenumber'),
-    standardFormat = phonenumber.PhoneNumberFormat.E164,
     CHARACTER_REGEX = /[a-z]/i;
 
 var _init = function(settings, phone) {
+  var instance = phonenumber.PhoneNumberUtil.getInstance();
+  var countryCode = settings && settings.default_country_code;
+  var regionCode = instance.getRegionCodeForCountryCode(countryCode);
+  var parsed = instance.parseAndKeepRawInput(phone, regionCode);
+
   return {
-    util: phonenumber.PhoneNumberUtil.getInstance(),
-    phone: phone,
-    countryCode: settings && settings.default_country_code,
-    country: function() {
-      return this.util.getRegionCodeForCountryCode(this.countryCode);
-    },
-    parse: function() {
-      return this.util.parseAndKeepRawInput(this.phone, this.country());
-    },
-    format: function() {
+    format: function(scheme) {
       if (!this.validate()) {
         return false;
       }
-      return this.util.format(this.parse(), standardFormat).toString();
+      if (typeof scheme === 'undefined') {
+        if (parsed.getCountryCode() + '' === countryCode) {
+          scheme = phonenumber.PhoneNumberFormat.NATIONAL;
+        } else {
+          scheme = phonenumber.PhoneNumberFormat.INTERNATIONAL;
+        }
+      }
+      return instance.format(parsed, scheme).toString();
     },
     validate: function() {
-      return this.util.isValidNumber(this.parse()) &&
+      return instance.isValidNumber(parsed) &&
         // Disallow alpha numbers which libphonenumber considers valid,
         // e.g. 1-800-MICROSOFT.
-        !this.phone.match(CHARACTER_REGEX);
+        !phone.match(CHARACTER_REGEX);
     }
   };
 };
 
-/** Returns international format if valid number, false if invalid number. */
+/**
+ * Returns international format if valid number, or false if invalid.
+ */
+exports.normalize = function(settings, phone) {
+  try {
+    return _init(settings, phone).format(phonenumber.PhoneNumberFormat.E164);
+  } catch (e) {}
+  return false;
+};
+
+/**
+ * Returns the number formatted for display, or false if invalid.
+ */
 exports.format = function(settings, phone) {
```

Comment:
`displayFormat` is a noun so doesn't work well for me. To me `format` and `normalize` definitely mean different things. Any other ideas?